backend/utils_segment_whole_slide.py

```diff
@@ -35,7 +39,7 @@ def get_saved_contours(level_array):
 def prepare_tile_info(
     x0: int, y0: int,
     w0: int, h0: int,
-    tile_size: int,
+    tile_size:  int,
```

Comment:
useless double space 
```suggestion
    tile_size: int,
```

Reply:
Deleted.